gt/main.c

```diff
@@ -1288,6 +1288,23 @@ process_gt_cmd(struct gt_cmd_entry *entry, struct gt_instance *instance)
 			rte_lcore_id());
 		break;
 
+	case GT_UPDATE_POLICY_INCREMENTALLY: {
+		/* Load the compiled Lua bytecode, and run it. */
+		int ret = luaL_loadbuffer(instance->lua_state,
+			entry->u.bc.lua_bytecode, entry->u.bc.len,
+			"bytecode") || lua_pcall(instance->lua_state, 0, 0, 0);
```

Comment:
1. Add `!= 0` to the returns of the functions to comply with DPDK's coding style.
2. Replace "bytecode" with "incremental_update_of_gt_lua_state".